source/guides/integrate-and-extend.rst

```diff
@@ -0,0 +1,40 @@
+Integrate and extend Mattermost
+================================
+
+.. toctree::
+  :maxdepth: 1
+  :hidden:
+  :titlesonly:
+
+    AI Copilot plugin </integrate/ai-copilot-plugin>
+    Channel Export plugin </integrate/export-mattermost-channel-data>
+    GitHub plugin </integrate/github-interoperability>
+    GitLab plugin </integrate/gitlab-interoperability>
+    Jira </integrate/jira-interoperability>
+    Legal hold
+    Performance metrics
+    Microsoft Teams </integrate/ms-teams-interoperability>
+    ServiceNow </integrate/servicenow-interoperability>
+    Zoom </integrate/zoom-interoperability>
+
+Mattermost offers many ways to add product functionality, custom-tailor your workflows, and connect Mattermost to your mission-critical tools. 
+
+Interoperability with pre-packaged integrations
+-----------------------------------------------
+
+Your Mattermost deployment comes with the following integrations you can configure and use:
+
+* :doc:`AI Copilot plugin </integrate/ai-copilot-plugin>` - Connect and use the Mattermost AI Copilot integration.
```

Comment:
Do we need an `Alpha` tag?

Reply:
AI Copilot isn't ready for release yet; will remove placeholders to it from this PR.